sdk/apimanagement/arm-apimanagement/review/arm-apimanagement.api.md

```diff
@@ -677,7 +677,7 @@ export class ApiManagementClient extends ApiManagementClientContext {
     beginPerformConnectivityCheckAsync(resourceGroupName: string, serviceName: string, connectivityCheckRequestParams: ConnectivityCheckRequest, options?: ApiManagementClientPerformConnectivityCheckAsyncOptionalParams): Promise<PollerLike<PollOperationState<ApiManagementClientPerformConnectivityCheckAsyncResponse>, ApiManagementClientPerformConnectivityCheckAsyncResponse>>;
     beginPerformConnectivityCheckAsyncAndWait(resourceGroupName: string, serviceName: string, connectivityCheckRequestParams: ConnectivityCheckRequest, options?: ApiManagementClientPerformConnectivityCheckAsyncOptionalParams): Promise<ApiManagementClientPerformConnectivityCheckAsyncResponse>;
     // (undocumented)
-    cache: Cache;
+    cache: Cache_2;
```

Comment:
I understand after reading your comments why these renames occur, but is the name collision with global interfaces a problem? My understanding is that being scoped to the module, they shouldn't be, but I wonder. Like, I probably wouldn't name a type `Window`, so I wonder if it's safe to have names that are the same as other common global types.

Reply:
I was chatting with @xirzec  about this - it's probably not ideal to shadow common global types and I believe API Extractor version upgrade will help surface where we do that today...